spec/routes/api/committee_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require_relative "spec_helper"
+
+RSpec.describe Clover, "committee infrastructure" do
+  let(:user) { create_account }
+  let(:project) { user.create_project_with_default_policy("project-1") }
+
+  before do
+    login_api(user.email)
+  end
+
+  it "serializes unexpected nested errors that cannot be converted" do
+    ex = Class.new(Committee::BadRequest) do
+      def self.name
+        "CommitteeTestNestedException"
+      end
+
+      def original_error
+        Exception.new("unaddressed error value")
+      end
+    end.new("testing conversion of unenumerated nested exceptions")
+
+    allow_any_instance_of(Committee::SchemaValidator::OpenAPI3).to receive(:request_validate) do
```

Comment:
If you want to avoid `allow_any_instance_of`, you could add a custom hash branch only in test that raises the error, similar to what we are doing here: https://github.com/ubicloud/ubicloud/blob/main/clover.rb#L468

Reply:
Maybe I can give that a try, although in some ways, I kinda like this, in the sense that the exactly one time such a test exception is injected, it's in the context of the single test that needs that; there are no others, there may never be another, and perhaps no more than one more other ever.